src/core/geometry_func_unittest.cpp

```diff
@@ -0,0 +1,32 @@
+/* $Id$ */
```

Comment:
SVN's been gone 18 months now, we should probably remove all these from the relevant files at some point... This is for a different PR though :)

Reply:
I created #7808 for this :)